app/views/help/cookie_settings.html.erb

```diff
@@ -18,31 +18,39 @@
   <%= render "govuk_publishing_components/components/govspeak", {
   } do %>
     <p>Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
-    <p>They store information about how you use the website, such as the pages you visit.</p>
-    <p>Cookies are used, for example, to:</p>
-    <ul>
-      <li>measure how you use the website so it can be updated and improved based on your needs</li>
-      <li>remember the notifications you've seen so we do not show them to you again</li>
-      <li>remember you while you’re changing your GOV.UK email subscriptions</li>
-    </ul>
-
-    <h2>Government services</h2>
-    <p>Most services we link to are run by different government departments, for example HM Revenue and Customs (HMRC) or the Home Office. These services may set additional cookies and, if so, will have their own cookies policy and banner linking to it.</p>
+    <p>They store information about how you use the GOV.UK website, such as the pages you visit.</p>
+
   <% end %>
 
   <%= render "govuk_publishing_components/components/heading", {
     text: t('cookies.cookie_settings'),
     padding: true,
     heading_level: 2
   } %>
-  <p>We use 4 main types of cookie. You can choose which cookies you're happy for us to use.</p>
+  <p>We use 4 types of cookie. You can choose which cookies you're happy for us to use.</p>
 
   <form data-module="cookie-settings">
+
+    <% cookies_usage_hint = capture do %>
```

Comment:
This text seems to be no longer being used? Is that intentional, as it's still in the Content doc